cJSON.h

```diff
@@ -81,84 +81,113 @@ typedef struct cJSON_Hooks
       void (*free_fn)(void *ptr);
 } cJSON_Hooks;
 
```

Comment:
gotcha.  The comment was above that block so in github UI you can't see what you are referring to.  Sure, no problem with this.